.github/workflows/update_tz_data.yaml

```diff
@@ -0,0 +1,65 @@
+name: Update eos-tzdata
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 1 * *"
+  push:
+    paths:
+      # if we change any of the non-generated items in tzdata, recheck
+      - "eos-tzdata/**"
+      - "!eos-tzdata/VERSION"
+      - "!eos-tzdata/src/data.rs"
+      - "!eos-tzdata/src/README//md"
+
+concurrency:
+  # since this runs on an interval this should be fine
+  # but we want concurrency failures since we're doing commits
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+
+jobs:
+  update-tz-data:
+    name: Update eos-tzdata using Python 3.x
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.x"
+
+      - name: Install Python deps
+        run: |
+          python -m pip install -U requests
+
+      - name: Revert last commit if it was GitHub Actions
+        run: |
+          if [[ "$(git show -s --format='%ae' HEAD)" == *"github-actions"* ]]; then
+            git reset --hard HEAD^;
+          fi
```

Comment:
Does this make sense if it's making a PR? I feel like it'd make a PR with a detached/out of date HEAD

Reply:
Agreed, I failed to remove this step in the last sweep. Got it now.